versions/3.1.0.md

```diff
@@ -2329,7 +2324,7 @@ In addition to the JSON Schema fields, the following OpenAPI vocabulary fields M
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema. Default value is `false`. Deprecated. Type arrays are now preferred, using the new `"null"` type, e.g: `"type": ["string", "null"]`.
+<a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema. Default value is `false`. Deprecated. Type arrays are now preferred, using the new `"null"` type, e.g: `"type": ["string", "null"]`. Nullable cannot be used in conjunction with a type array and implementations MUST ignore it.
```

Comment:
How about:

> <a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema when used alongside the `type` keyword with a single value. Default value is false.  Deprecated. Type arrays are now preferred, using the `"null"` type alongside one or more other types, e.g: `"type": ["string", "null"]`. `nullable` cannot be used in conjunction with a type array, or when `type` is absent, and implementations MUST ignore it in such cases.  In particular, the default value of `nullable` MUST be ignored when `type` is absent or when `type` has an array value such as `["string", "null"]`.

This will ensure that the empty schema, `{}`, correctly allows the JSON `null` literal as a valid value.  To do otherwise violates a fundamental design principle of JSON Schema, and there is no way we can claim compatibility.  The empty schema MUST NOT ever produce an invalid result.